crates/sui-package-management/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Minting this crate to perform package management logic in the lock file. Will grow, e.g., to handle logic for detecting conflicts (e.g., if `Move.toml` specifies `published-at` address versus `Move.lock` automated info) and resolution for `publish`/`upgrade`.